src/components/RightSidebar/EditButton.astro

```diff
@@ -0,0 +1,41 @@
+---
+interface Props {
+	editHref: string,
+	iconLeft: boolean,
+	bold: boolean
+}
+import UIString from '../UIString.astro';
+const { editHref, iconLeft = true, bold = false } = Astro.props;
+const fontWeight = bold ? "bold" : "normal";
+---
+<a class="edit-on-github" href={editHref} target="_blank">
+	{!iconLeft && <span><UIString key="rightSidebar.editPage" /></span>}
+	<svg
+		aria-hidden="true"
+		focusable="false"
+		data-prefix="fas"
+		data-icon="pen"
+		class="svg-inline--fa fa-pen fa-w-16"
+		role="img"
+		xmlns="http://www.w3.org/2000/svg"
+		viewBox="0 0 512 512"
+		height="1em"
+		width="1em"
+	>
+		<path
+			fill="currentColor"
+			d="M290.74 93.24l128.02 128.02-277.99 277.99-114.14 12.6C11.35 513.54-1.56 500.62.14 485.34l12.7-114.22 277.9-277.88zm207.2-19.06l-60.11-60.11c-18.75-18.75-49.16-18.75-67.91 0l-56.55 56.55 128.02 128.02 56.55-56.55c18.75-18.76 18.75-49.16 0-67.91z"
+		></path>
+	</svg>
+	{iconLeft && <span><UIString key="rightSidebar.editPage" /></span>}
+</a>
+
+<style define:vars={{fontWeight}}>
+	.edit-on-github {
```

Comment:
I’m not quite sure about the way this is styled in this context.

<img width="578" alt="image" src="https://user-images.githubusercontent.com/357379/198284340-3e7fefc8-7728-46e0-b354-0a94946a9784.png">

Our other links in the main content all follow the same styling, so I wonder plain link styling is all we need here — maybe without the icon too which I find sits slightly oddly in a paragraph like this.

I’m realising this is also partially an `<Aside>` issue because it changes our default link color, so without custom styling it’s just:

<img width="564" alt="image" src="https://user-images.githubusercontent.com/357379/198284786-bf4db0d8-b0d1-4c7a-a015-c8c8f9ef5314.png">

Maybe we just want it to be bold?

<img width="577" alt="image" src="https://user-images.githubusercontent.com/357379/198288501-15f35c4e-b6b9-4adc-af63-2ec56485a2d0.png">

The other option would be to go fully in on making this look like a button, but I’m not sure we want to create yet another UI variant just yet?

<img width="578" alt="image" src="https://user-images.githubusercontent.com/357379/198289046-f601fe9e-a265-44d9-af92-5c698ac9e7d6.png">

Reply:
Went with the bold!